ghost/core/core/server/services/members/utils.js

```diff
@@ -4,6 +4,7 @@ function formatNewsletterResponse(newsletters) {
     });
 }
 
+module.exports.formatNewsletterResponse = formatNewsletterResponse;
```

Comment:
This is the meat of the change. We use the default newsletter formatter for the rest of the member endpoints.